app/assets/javascripts/media_player_wrapper/avalon_player_new.es6

```diff
@@ -330,7 +330,12 @@ class MEJSPlayer {
 
     // Grab instance of player
     if (!this.player) {
-      this.player = this.mediaElement;
+      this.player = instance;
+    }
+
+    if (this.player && this.player.media && this.player.media.hlsPlayer && this.player.media.hlsPlayer.config) {
+      // Workaround for hlsError bufferFullError: Set max buffer length to 2 minutes
+      this.player.media.hlsPlayer.config.maxMaxBufferLength = 120;
     }
```

Comment:
Fixes #677: on Chrome, media player stops partway due to HLS `bufferFullError`, this fix sets a maxMaxBufferLength in the HLS player.